packages/service/src/account/live-account-service.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    CustomHttpHeaders,
+    OperationOptions,
+    getArmUrl,
+    getCacheManager,
+} from "@azure/bonito-core";
+import { BatchAccountOutput } from "../arm-batch-models";
+import { AccountService } from "./account-service";
+import { createARMBatchClient, isUnexpected } from "../internal/arm-batch-rest";
+import {
+    createArmUnexpectedStatusCodeError,
+    parseBatchAccountIdInfo,
+} from "../utils";
+
+export class LiveAccountService implements AccountService {
+    async get(
+        accountResouceId: string,
+        opts?: OperationOptions
+    ): Promise<BatchAccountOutput> {
+        const { subscriptionId, resourceGroupName, batchAccountName } =
+            parseBatchAccountIdInfo(accountResouceId);
+        const cacheManager = getCacheManager();
+        const armBatchClient = createARMBatchClient({
+            baseUrl: getArmUrl(),
+        });
+
+        const cacheKey = `get-${accountResouceId}`;
```

Comment:
How does this cache ever get invalidated? Every Batch account we ever load will be stored in memory until a browser refresh. For accounts this isn't likely to be a huge issue, but it is a memory leak.

Reply:
We don't necessarily need to add a TTL, and I don't think we need to fully clear the cache either, but leaking every account that was accessed isn't great. Can we do something like an LRU cache with a fixed size for now? That might not scale to all of the ways we might use the cache in the future, but it seems fine for now.

Reply:
I just pushed a changes which includes a LRU cache implementation. The LRU cache just record the which key is being used and delete the least recent used cache when the capacity exceeds.